### PR TITLE
Make build input urls secret so they can be used with secure feeds.

### PR DESCRIPTION
**Pull request by mmitche**

### CODE REVIEW

**Comment by chcosta** [comment]
I know we briefly chatted about this in the morning, but I don't recall the details.  Where are the values for these properties coming from?  ie, if it's not coming from key vault, then there's the possibility that someone could unintentionally mess up the "isSecret" part and leak a secret.  Is that the case here?

File: buildpipeline/DotNet-CoreFx-Trusted-OSX.json [hunk 1]
```diff
@@ -79,7 +79,9 @@
       }
     },
     {
-      "environment": {},
+      "environment": {
+        "PACKAGEVERSIONPROPSURL": "$(PB_PackageVersionPropsUrl)"
```
Comment (weshaggard) [inline buildpipeline/DotNet-CoreFx-Trusted-OSX.json]: Did this work?

**Comment by mmitche** [comment]
@chcosta They come from keyvault

File: buildpipeline/DotNet-CoreFx-Trusted-OSX.json (hunk 1 above)
Comment (mmitche) [reply buildpipeline/DotNet-CoreFx-Trusted-OSX.json]: Yep!  I confirmed with VSTS that this is the way to do it.

**Pull request merged**